### ISSUE TITLE
Sometimes cargo-watch crashes when a test fails

### ISSUE THREAD

**Issue by paulkernfeld** [passcod/cargo-watch#61]
I'm guessing that the desired behavior for `cargo-watch` is to keep running even if a test fails in a weird way. I've seen a few cases where `cargo-watch` crashes when a test fails.

From running `cargo watch -x test`, I see the following logs:

```
...
error: build failed
ERROR:cargo_watch::schedule: Error trying to check status of job, abort.
ERROR:cargo_watch::schedule: command ["/bin/sh", "-c", "cargo test"] exited with code 101
```

**Comment by passcod** [comment]
Can you upgrade to the latest? The message you're seeing was changed (and fixed) in [4.0.1](https://github.com/passcod/cargo-watch/releases/tag/v4.0.1).

**Comment by paulkernfeld** [comment]
Great! Thanks very much!
